util/sorc/compile_gfs_util_wcoss.sh

```diff
@@ -10,7 +10,7 @@ LMOD_EXACT_MATCH=no
 source ../../sorc/machine-setup.sh > /dev/null 2>&1
 cwd=`pwd`
 
-if [ "$target" = "wcoss_dell_p3" ] || [ "$target" = "wcoss_cray" ] || [ "$target" = "hera" ] ; then
+if [ "$target" = "wcoss_dell_p3" ] || [ "$target" = "wcoss_cray" ] || [ "$target" = "hera" ] || [ "$target" = "wcoss2" ] ; then
```

Comment:
This line is checking for `hera` and the echo statement says `You are on WCOSS: $target`

Reply:
Removed Hera reference